chapter_convolutional-neural-networks/channels.md

```diff
@@ -142,12 +162,25 @@ K = d2l.normal((2, 3, 1, 1), 0, 1)
 ```
 
 ```{.python .input}
-#@tab all
+#@tab paddle
```

Comment:
consider reusing the code.

Reply:
I tried to use the code of d2l package to modify it, but there was an error, so I kept it here and made some small changes.